jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -84,53 +81,89 @@ The following details the specifications and APIs currently planned for the Jaka
-* Jakarta Bean Validation 3.0 (Web Profile)
-* Jakarta Concurrency 3.0* (Web Profile)
-* Jakarta Connectors 2.1*
-* Jakarta Contexts and Dependency Injection 4.0*
-* Jakarta Debugging Support for Other Languages 2.0 (Web Profile)
-* Jakarta Dependency Injection 2.0 (Web Profile)
-* Jakarta Enterprise Beans 4.0
-* Jakarta Enterprise Beans 4.0 Lite (Web Profile)
-* Jakarta Enterprise Web Services 2.0 (Optional)
-* Jakarta Expression Language 5.0* (Web Profile)
-* Jakarta Interceptors 2.1* (Web Profile)
-* Jakarta JSON Binding 3.0* (Web Profile)
-* Jakarta JSON Processing 2.1* (Web Profile)
-* Jakarta Mail 2.1*
-* Jakarta Managed Beans 2.0 (Web Profile)
-* Jakarta Messaging 3.1*
-* Jakarta Persistence 3.1* (Web Profile)
-* Jakarta RESTful Web Services 3.1* (Web Profile)
-* Jakarta Security 3.0* (Web Profile)
-* Jakarta Server Faces 4.0* (Web Profile)
-* Jakarta Server Pages 3.1* (Web Profile)
-* Jakarta Servlet 6.0* (Web Profile)
-* Jakarta SOAP with Attachments 3.0* (Optional)
-* Jakarta Standard Tag Library 3.0* (Web Profile)
-* Jakarta Transactions 2.0 (Web Profile)
-* Jakarta WebSocket 2.1* (Web Profile)
-* Jakarta XML Binding 4.0* (Optional)
-* Jakarta XML Web Services 4.0* (Optional)
-* Jakarta Web Services Metadata 3.0 (Merged)
--->
+- Jakarta EE Platform 11*
+- Jakarta EE Web Profile 11*
+- Jakarta EE Core Profile 11*
+
+- Jakarta "CDI implementation spec" 4.1*
+- Jakarta "CDI integration spec" 4.1*
+- Jakarta Activation 2.1.2* service release
+- Jakarta Annotations 3.0*
+- Jakarta Authentication 3.1*
+- Jakarta Authorization 3.0*
+- Jakarta Batch 2.1
+- Jakarta Concurrency 3.1*
+- Jakarta Connectors 2.1
+- Jakarta Debugging Support for Other Languages 2.0
+- Jakarta Dependency Injection 2.0.1
+- Jakarta Enterprise Beans 4.0.1
+- Jakarta Expression Language 4.0
+- Jakarta Faces 5.0*
+- Jakarta Interceptors 2.2*
+- Jakarta JSON Binding 3.0
+- Jakarta JSON Processing 2.1.2* service release
+- Jakarta Lang Model 4.0.1
+- Jakarta Mail 2.1.2* service release
+- Jakarta Messaging 3.1
+- Jakarta Persistence 3.2*
+- Jakarta RESTful Web Services 4.0*
+- Jakarta SOAP with Attachments 3.0
+- Jakarta Security 4.0*
+- Jakarta Server Pages 4.0*
+- Jakarta Servlet 6.1*
+- Jakarta Standard Tag Library 3.0
+- Jakarta Transactions 2.0.1
+- Jakarta Validation 3.1*
+- Jakarta WebSocket 2.2*
+- Jakarta XML Binding 4.0
+- Jakarta XML Web Services 4.0
+
+### Proposed Updates to Platform
+
+- Jakarta "CDI integration spec"
+- Jakarta Activation
+- Jakarta Annotations
+- Jakarta Authentication
+- Jakarta Authorization
+- Jakarta Concurrency
+- Jakarta Faces
+- Jakarta Interceptors
+- Jakarta JSON Processing
+- Jakarta Mail
+- Jakarta Persistence
+- Jakarta RESTful Web Services
+- Jakarta Security
+- Jakarta Server Pages
+- Jakarta Servlet
+- Jakarta Transactions
+- Jakarta Validation
+- Jakarta WebSocket
 
 ### Proposed Updates to Web Profile
 
+- Jakarta Annotations
+- Jakarta Validation
+- Jakarta Concurrency
+- Jakarta "CDI integration spec"
```

Comment:
How about Jakarta CDI for Enterprise

Reply:
See https://github.com/jakartaee/jakartaee-platform/pull/729#discussion_r1285279491